doc/vaulted-edit.1.md

```diff
@@ -17,3 +17,10 @@ DESCRIPTION
 Spawns an interactve mode for editing the content of an existing vault.
 
 Upon quitting, the new content is saved to the vault.
+
+AWS KEY
+-------
+
+*Substituted credentials:* Can be toggled to have your AWS credentials
```

Comment:
After looking at the documentation this generates, I'm less convinced this is the way to go. How about we use a bulleted list of menu options, like so:

```
 * k - Key
 * m - MFA
 * r - Role
 * c - Substitute with temporary credentials  
   Toggles whether your AWS credentials are substituted with a set of temporary
   credentials. For more details on this process, see the documentation for
   `vaulted shell`.
 * s - Show Key
 * D - Delete
```

(also, I reworded the message a little, see what you think)